Ska/Sun/tests/test_sun.py

```diff
@@ -3,7 +3,18 @@
 import numpy as np
 from Quaternion import Quat
 from ..sun import (apply_sun_pitch_yaw, get_sun_pitch_yaw, nominal_roll,
-                   off_nominal_roll, position, pitch as sun_pitch)
+                   off_nominal_roll, position, allowed_rolldev,
+                   pitch as sun_pitch)
+
+
+def test_allowed_rolldev():
+    assert np.isclose(allowed_rolldev(135), 13.979)
```

Comment:
There should be one case which is fine-tuned to land between a steep transition (at the throat) to prove that the `minimum` business is working.

Reply:
Also add a test with an array input. You can just repeat all of the scalar values.